src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import javax.swing.text.DateFormatter;
+
+public class Deadline extends Task{
+    private LocalDate by;
```

Comment:
Although this code is just a WIP, I would just like to leave a comment: perhaps changing out the private LocalDate "by" with dateBy, or a more descriptive name would be better! I realized that you use isDone and is... for your boolean variables, so that's good, and your other variables should also reflect that level of precise descriptiveness (in camelcase, of course.)

Reply:
Thanks so much for reading! Hope the (somewhat wordy) comments has been somewhat useful. 😄